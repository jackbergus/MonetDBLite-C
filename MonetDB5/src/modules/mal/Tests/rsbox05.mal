# Simple test that tests whether `unknown atoms' are currectly printed
# as strings

include rsbox;

b := bat.new(:void, :date);
bat.insert(b, nil, "1999-05-02":date);

rs := rsbox.create(1, 't');
resultset.populateColumn(rs, 1, b);

resultset.writeNextPart(rs);
