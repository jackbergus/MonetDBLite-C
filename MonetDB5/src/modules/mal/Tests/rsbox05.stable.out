stdout of test 'rsbox05` in directory 'src/modules/mal` itself:


# 20:14:44 >  
# 20:14:44 >  Mtimeout -timeout 60 mserver5 "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib64/MonetDB5:/ufs/mk/monet5/Linux/lib64/MonetDB5/lib:/ufs/mk/monet5/Linux/lib64/MonetDB5/bin" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/dblogs" --set mapi_port=35008 --set monet_prompt= --trace --dbname=mTests_src_modules_mal  rsbox05.mal < /dev/null
# 20:14:44 >  

# MonetDB Server v4.99.19
# Copyright (c) 1993-2006 CWI, all rights reserved
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs dynamically linked
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_modules_mal
# Visit http://monetdb.cwi.nl/ for further information
## Simple test that tests whether `unknown atoms' are currectly printed
## as strings
#
#include rsbox;
#
#b := bat.new(:void, :date);
#bat.insert(b, nil, "1999-05-02":date);
#
#rs := rsbox.create(1, 't');
#resultset.populateColumn(rs, 1, b);
#
#resultset.writeNextPart(rs);
#
function user.main():void;	# 0  (main:void)
# Simple test that tests whether `unknown atoms' are currectly printed 	# 1  (_1:str)
# as strings 	# 2  (_2:str)
    b := bat.new(:void,:date);	# 3 CMDBATnew (b:bat[:void,:date])<-(_4:void)(_5:date)
    bat.insert(b,nil:void,1999-05-02:date);	# 4 BKCinsert_bun (_6:void)<-(b:bat[:void,:date])(_7:void)(_8:date)
    rs := rsbox.create(1,'t');	# 5 RSBcreate (rs:resultset)<-(_10:int)(_11:chr)
    resultset.populateColumn(rs,1,b);	# 6 RSpopulateColumn (_12:resultset)<-(rs:resultset)(_10:int)(b:bat[:void,:date])
    resultset.writeNextPart(rs);	# 7 RSwriteNextPart (_13:int)<-(rs:resultset)
end main;	# 8  
&t
$id	1
$columncount	1
$tuplecount	1
$ctype	S
%column	column_1
%table	
%schema	
%type	varchar
%digits	
%scale	
%width	0
[10:1999-05-02

# 20:14:44 >  
# 20:14:44 >  Done.
# 20:14:44 >  

